test/bazel/proto/BUILD.bazel

```diff
@@ -17,3 +17,8 @@ typescript_proto_library(
     name = "pizza_service_ts_proto",
     proto = ":pizza_service_proto",
 )
+
+typescript_proto_library(
```

Comment:
This isn't really a test since right now the only thing that's checked in the CI is if this rule builds. I guess it's OK, but it would be better to use this proto in a web test or check that the file is where you expect it to be (e.g. in an sh_test).

Reply:
Can you be more specific? I don't see the sh_tests you're referring to. And are you suggesting writing a test like [`pizza_service_proto_test.spec.ts`](https://github.com/improbable-eng/ts-protoc-gen/blob/master/test/bazel/pizza_service_proto_test.spec.ts)?

Reply:
My bad, let me say it a different way: It would be better to use the generated file explicitly in a test. Two ways you could do this are (I'm suggesting do one):

1. Use the file in a new test, just like `pizza_service_proto_test.spec.ts`
2. Write a test that reads the file and checks the contents. This could be implemented in many ways, an [sh_test](https://github.com/bazelbuild/bazel/blob/master/src/test/shell/bazel/BUILD#L66) is just an example.

#-1 is probably easier since there are existing tests you can base the test off of.